susemanager-utils/susemanager-sls/src/beacons/pkgset.py

```diff
@@ -20,7 +20,8 @@ def __virtual__():
         os.path.exists("/usr/lib/zypp/plugins/commit/susemanager") or  # Remove this once 2015.8.7 not in use
         os.path.exists("/usr/lib/zypp/plugins/commit/zyppnotify") or
         os.path.exists("/usr/share/yum-plugins/susemanagerplugin.py") or  # Remove this once 2015.8.7 not in use
-        os.path.exists("/usr/share/yum-plugins/yumnotify.py")
+        os.path.exists("/usr/share/yum-plugins/yumnotify.py") or
+        os.path.exists("/usr/bin/dpkgnotify")
```

Comment:
When you are already here: Please remove the two lines which say:

   # Remove this once 2015.8.7 not in use

:-)

Reply:
ahah good point. I will add (or in this case remove) in another commit